pandas/core/groupby/groupby.py

```diff
@@ -444,14 +444,15 @@ class providing the base-class of operations.
     a `(callable, data_keyword)` tuple where `data_keyword` is a
     string indicating the keyword of `callable` that expects the
     %(klass)s object.
-args : iterable, optional
+*args : iterable, optional
        Positional arguments passed into `func`.
-kwargs : dict, optional
+**kwargs : dict, optional
          A dictionary of keyword arguments passed into `func`.
 
 Returns
 -------
-the return type of `func`.
+object
```

Comment:
I think this should be `%(klass)s`, if I'm not wrong `DataFrame.pipe` returns a `DataFrame` and `Series.pipe` returns a `Series`.

Reply:
Agreed! Thanks for the tip and the explanation. Updating the PR with this change.